discovery-provider/src/queries/get_health.py

```diff
@@ -157,6 +147,18 @@ def get_health(args, use_redis_cache=True):
         latest_indexed_block_num = db_block_state["number"] or 0
         latest_indexed_block_hash = db_block_state["blockhash"]
 
+    # Get system information monitor values
+    sys_info = monitors.get_monitors([
+        MONITORS['DATABASE_SIZE'],
```

Comment:
is REDIS_TOTAL_MEMORY returned in a query somewhere else...?

Reply:
atm, not returned, but since we added it to the content node, i thought it would be useful to have available. added it here for now as well.

Reply:
sweet yeah i thought itwas a useful metric!